### ISSUE TITLE
Unable to locate package endaga-osmocom

### ISSUE THREAD

**Issue by emersonjr** [facebookincubator/CommunityCellularManager#66]
I'm following the client instructions. Until now i issued.
 ```
vagrant up osmocom
vagrant ssh osmocom
apt-get install endaga-osmocom
 ```
When i issue `apt-get install endaga-osmocom` i get this:
```
root@endaga-client-osmocom:/home/vagrant# apt-get install endaga-osmocom
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package endaga-osmocom
root@endaga-client-osmocom:/home/vagrant# 
```

I've already issued `apt-get update`. The output of `apt-cache search endaga` is:

```
root@endaga-client-osmocom:/home/vagrant# apt-cache search endaga
python-endaga-core - Endaga client core
python-pynexmo - Python library for nexmo, with some updates from Endaga. Forked from https://github.com/marcuz/libpynexmo
root@endaga-client-osmocom:/home/vagrant# 
```

**Comment by kkroo** [comment]
Hey @emersonjr,
Can you check that you are tracking `deb http://repo.etagecom.io dev main` within `/target/etc/apt/sources.list.d/`?
Keep me posted
Omar

**Comment by emersonjr** [comment]
Hi @kkroo 
I just destroyed and did `vagrant up osmocom` again then i could install endaga-osmocom seamlessly.
I think the provisioning of VM wasn't made properly that first time. Now it's ok.
I did `ls /etc/apt/sources.list.d/` on this working VM now and there's a `20_repo_etagecom_io.list`, into it there's the following content:
```
deb http://repo.etagecom.io dev main
deb http://repo.etagecom.io test main
deb http://repo.etagecom.io beta main
```

It seems to be good for now, thank you.
